proposals/core.md

```diff
@@ -116,40 +116,36 @@ To use the new bidirectional protocol, the client must first establish a WebSock
 
 >[RESOLUTION](https://www.w3.org/2019/09/20-webdriver-minutes.html#resolution01): Bi-di is always enabled. An optional capability, defaulting to true, indicating that bi-di is desired. When a new session is established, the return value of the new session contains the new top-level property of the bi-directional URL

```

Comment:
Note that what we landed in https://github.com/w3c/webdriver-bidi/pull/24 doesn't default to true. Should we revisit that?

Reply:
Defaulting to true seems wrong to me. Once support is there clients that are using it can start to default to true.

Reply:
It seems wrong to me as well. Reading the conversation leading up to https://www.w3.org/2019/09/20-webdriver-minutes.html#resolution01 I can't find any clues about whether this was deliberate or not, and I wasn't at these meetings. I guess I'll just leave the contradiction here and maybe ask about it quickly in a future VC.